packages/terra-alert/tests/jest/Alert.test.jsx

```diff
@@ -10,12 +10,14 @@ import IconInformation from 'terra-icon/lib/icon/IconInformation';
 import IconSuccess from 'terra-icon/lib/icon/IconSuccess';
 import IconWarning from 'terra-icon/lib/icon/IconWarning';
 import Button from 'terra-button';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import { v4 as uuidv4 } from 'uuid';
 import Alert from '../../src/Alert';
 
+const mockUuid = '00000000-0000-0000-0000-000000000000';
 let mockSpyUuid;
 beforeAll(() => {
-  mockSpyUuid = jest.spyOn(uuidv4, 'v4').mockReturnValue('00000000-0000-0000-0000-000000000000');
+  mockSpyUuid = jest.spyOn(uuidv4, 'v4').mockReturnValue(mockUuid);
```

Comment:
We should hardcode this otherwise, it may show up as undefined when testing in other components.

Reply:
Will undo these!

Reply:
Updated here https://github.com/cerner/terra-core/pull/3908/commits/4678cc76163c88a262dc1483d6a4489b1ab4dc42